scprep/run/dyngen.py

```diff
@@ -0,0 +1,241 @@
+           
+    num_targets: int, optional (default: 50)
+           Number of target genes. Target genes are regulated by a TF or another target gene, but are always downstream of at least one TF. 
+    num_hks: int, optional (default: 25)
+           Number of housekeeping genees. Housekeeping genes are completely separate from any TFs or target genes.
+    simulation_census_interval: int, optional (default: 10)
+           Allows storing the abundance levels of molecules only after a specific interval has passed since the previous census. By setting the census interval to 0, the whole simulation’s trajectory is retained but many of these time points will contain very similar information.
+    compute_cellwise_grn: boolean, optional (default: False)
+           Whether or not to compute the ground truth cellwise gene regulatory networks after simulation. Also outputs ground truth bulk (entire dataset) regulatory network. Increases compute time significantly.
+    compute_rna_velocity: boolean, optional (default: False)
+           Whether or not to compute the ground truth propensity ratios after simulation. Increases compute time significantly.
+    n_jobs: int, optional (default: 8)
+           Number of cores to use.
+    random_state: int, optional (default: None)
+           Fixes seed for simulation generator.
+    lib_loc: string, optional (default: None)
+           Library directory from which to load dyngen.
+           If None, defaults to the first element of .libPaths()
+    verbose: boolean, optional (default: True)
+           Data generation verbosity.
+             
+    Returns
+    -------
+    Dictionary data of pd.DataFrames:
+    ...data['cell_info']: pd.DataFrame, shape (n_cells, 7)
+           Columns: cell_id, step_ix, simulation_i, sim_time, num_molecules, mult, lib_size
+           sim_time is the simulated timepoint for a given cell.
+          
+    ...data['expression']: pd.DataFrame, shape (n_cells, n_genes)
+           Log-transformed counts with dropout.
+       
+    If compute_cellwise_grn is True,
+    ...data['bulk_grn']: pd.DataFrame, shape (n_tf_target_interactions, 4)
+           Columns: regulator, target, strength, effect.
+           Strength is positive and unbounded, and effect is either +1 (for activation) or -1 (for inhibition)
+           
+    ...data['cellwise_grn']: pd.DataFrame, shape (n_tf_target_interactions_per_cell, 4)
+           Columns: cell_id, regulator, target, strength. 
+           The output does not include all edges per cell. The regulatory effect lies between [−1, 1], where -1 represents complete inhibition of target by TF, +1 represents maximal activation of target by TF, and 0 represents inactivity of the regulatory interaction between R and T.
+       
+    If compute_rna_velocity is True,
+    ...data['rna_velocity']: pd.DataFrame, shape (n_cells, n_genes)
+           Propensity ratios for each cell.
+
+    Example
+    --------
+    scprep.run.dyngen.install()
```

Comment:
Format `Examples` as in https://github.com/KrishnaswamyLab/scprep/blob/37a1c924199c838bd60276b88bdb2b0fec04df94/scprep/stats.py#L42